lblod/app-worship-decisions-database
# Adjusting report to include Gemeenten/Provincies

# Description

DL-5469

This PR adds extra administrative units in the report `services-without-email-notifications.js` :
- Change `worship-services-without-email-notifications.js` (Removed)  to  `services-without-email-notifications.js`
- Adjusting the report to include Gemeenten and Provincies (Query change + Paths)

# Type of change


# Related services

- N/A

# How to test 

1. Setup in docker-compose.override.yml + run the stack
```yml
  report-generation:
    links:
      - virtuoso:database  # we need to because performance
    ports:
      - 9999:80
```
2. Run curl to generate report
```bash
curl -X POST \               
  -H "Content-Type: application/vnd.api+json" \
  -d '{
    "data": {
      "attributes": {
        "reportName": "servicesWithoutEmailNotifications"
      }
    }
  }' \
  http://localhost:9999/reports
```
3. You should be able to see a generated report in `data/files/services-without-email-notifications-3e6a0140-854b-11ee-9375-8b71f5f717ea` (uuid is randomized), note you can also check the report in dashboard/report

# What to check

- N/A 

# Links to other PR's

- N/A

# Notes

- N/A
